moped-editor/src/views/staff/StaffListView.js

```diff
@@ -1,73 +1,112 @@
-import React, { useState } from "react";
+import React from "react";
 
-import { FormControlLabel, FormGroup, Switch } from "@mui/material";
-import makeStyles from '@mui/styles/makeStyles';
-import Page from "src/components/Page";
+import { Box, Card, Container, CircularProgress } from "@mui/material";
+import makeStyles from "@mui/styles/makeStyles";
+import { useQuery } from "@apollo/client";
+import { DataGrid, GridToolbar } from "@mui/x-data-grid";
 
-import { StaffListViewQueryConf } from "./StaffListViewQueryConf";
-import GQLAbstract from "../../libs/GQLAbstract";
-import GridTable from "../../components/GridTable/GridTable";
+import Page from "src/components/Page";
+import { GET_ALL_USERS } from "src/queries/staff";
+import { AddUserButton, EditUserButton } from "./StaffListViewCustomComponents";
 
 const useStyles = makeStyles((theme) => ({
   root: {
     minHeight: "100%",
     paddingBottom: theme.spacing(3),
     paddingTop: theme.spacing(3),
   },
-  switch: {
-    marginTop: "1rem",
+  container: {
+    maxWidth: "100%",
+  },
+  addStaffButton: {
+    display: "flex",
     justifyContent: "flex-end",
-    alignItems: "flex-end",
+    marginBottom: "24px",
   },
 }));
 
 const StaffListView = () => {
   const classes = useStyles();
 
-  const [showInactive, setShowInactive] = useState(false);
+  const { data, loading, error } = useQuery(GET_ALL_USERS);
+
+  const staffColumns = [
```

Comment:
i _think_ because these column definitions are static we can move them outside the component?